docs/resources/resources-glossary.md

```diff
@@ -0,0 +1,132 @@
+
+## requirement set
+
+[!include[Requirement set note](../includes/office-js-requirement-sets.md)]
+
+## ribbon, ribbon button
+
+A **ribbon** is a command bar that organizes an application's features into a series of tabs or buttons at the top of a window. A **ribbon button** is one of the buttons within this series.
+
+## runtime
+
+In the Office Add-ins context, a **runtime** is a lifecycle, or the time during which an application is running.
+
+## task pane
+
+Task panes are interface surfaces that typically appear on the right side of the window within Word, PowerPoint, Excel, and Outlook. Task panes give users access to interface controls that run code to modify documents or emails, or display data from a data source. Use task panes when you don't need to embed functionality directly into the document. See [Task panes in Office Add-ins](../design/task-pane-add-ins.md) to learn more.
+
+## tutorial
+
+A **tutorial** is a teaching aid designed to help people learn to use a product or procedure. In the Office Add-ins context, a tutorial guides an add-in developer through the complete add-in development process for a particular application, such as Excel. This involves following 20 or more steps and is a greater time investment than a [quickstart](#quickstart).
+
+See also: [quickstart](#quickstart).
+
+## quickstart
+
+A **quickstart** is a a high level description of key skills and knowledge required for the basic operation of a particular program. In the Office Add-ins documentation, a quickstart is an introduction to developing an add-in for a particular application, such as Outlook. A quickstart contains series of steps that an add-in developer can complete in approximately 5 minutes, resulting in a functioning add-in.
+
+See also: [tutorial](#tutorial).
+
+## UI-less custom function
+
+A **UI-less custom function** is a custom functions add-in that doesn't have a task pane or other user-interface elements.
+
+See also: [custom function](#custom-function).
+
+## web add-in
+
+In the Office Add-ins documentation, **web add-in** is a legacy term for Office Add-ins.
+
+See also: [add-in](#add-in).
+
+## XLL
+
+An **XLL** add-in is an Excel add-in file with the file extension **.xll**. An XLL file is a type of dynamic link library (DLL) file that can only be opened by Excel. XLL add-in files must be written in C or C++. See [Developing Excel XLLs](/office/client-developer/excel/developing-excel-xlls) to learn more.
+
+See also: [custom function](#custom-function).
+
+## Yeoman generator, yo office
+
+The [Yeoman generator for Office Add-ins](https://github.com/OfficeDev/generator-office) uses the open source [Yeoman](https://github.com/yeoman/yo) tool to generate an Office Add-in via the command line. **Yo office** is the command line argument that runs the Yeoman generator for Office Add-ins. The Office Add-ins quickstarts and tutorials use the Yeoman generator.
```

Comment:
Since this refers to the command line argument, consider stylizing the casing accordingly.
```suggestion
The [Yeoman generator for Office Add-ins](https://github.com/OfficeDev/generator-office) uses the open source [Yeoman](https://github.com/yeoman/yo) tool to generate an Office Add-in via the command line. **yo office** is the command line argument that runs the Yeoman generator for Office Add-ins. The Office Add-ins quickstarts and tutorials use the Yeoman generator.
```